tests/lib/rules/comma-dangle.js

```diff
@@ -271,7 +271,7 @@ ruleTester.run("comma-dangle", rule, {
             options: ["always-multiline"],
         },
         {
-            code: "foo(\na,\mb\m)",
```

Comment:
Is this supposed to be something different? Removing only the useless escapes would result in

```js
    code: "foo(\na,mbm)",
```

But I'm not sure what the intention of the test is.